src/Illuminate/Log/SingleChannel.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Illuminate\Log;
+
+use Monolog\Handler\StreamHandler;
+
+class SingleChannel extends Channel
+{
+    /**
+     * The default options for the channel.
+     *
+     * @var array
+     */
+    protected $defaultOptions = [
+        'file' => 'laravel.log',
+    ];
+
+    /**
+     * Setup a new handler for a single file.
+     *
+     * @param  array  $options
+     * @return void
+     */
+    public function prepare(array $options = [])
+    {
+        $options = array_merge($this->defaultOptions, $options);
+
+        $this->writer->pushHandler(new StreamHandler($this->app->storagePath().'/logs/'.$options['file']));
```

Comment:
In production we leave our logs files outside Laravel storage path. Our deployment swaps the whole application folder to a symlink, so make it possible to use any path is better to hardcoded storage path.

Reply:
good shout